src/scheduler/listNotebookScheduler.tsx

```diff
@@ -288,7 +291,7 @@ function listNotebookScheduler({
   };
 
   const handleDeleteImportError = async (dagId: string) => {
-    const fromPage = "importErrorPage"
+    const fromPage = 'importErrorPage';
```

Comment:
Let's keep the quote style consistent throughout the file. The rest of the string literals use double quotes, so this line should do that too.